.github/FUNDING.yml

```diff
@@ -1,13 +1,12 @@
 # These are supported funding model platforms
-
 github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
 patreon: # Replace with a single Patreon username
 open_collective: # Replace with a single Open Collective username
 ko_fi: martinet101
 tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
-community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+community_bridge: # Replace with a single Community Bridge project name e.g., cloud-foundry
```

Comment:
This is documentation GitHub adds to funding.yml by default. I reckon if it has the dash, it must be for something. What do you think?